src/main/java/net/atos/zac/formulieren/model/FormulierVeldDefinitie.java

```diff
@@ -51,7 +52,7 @@ public class FormulierVeldDefinitie {
     @NotNull
     @Column(name = "veldtype", nullable = false)
     @Enumerated(EnumType.STRING)
-    private FormulierVeldType type;
+    private FormulierVeldType veldType;
```

Comment:
"veldtype" is 1 woord. Dus dit mag gewoon 'veldtype' heten. de column heet ook velddtype